### ISSUE TITLE
Connection errored out without error

### ISSUE THREAD

**Issue by Syncrodus** [cocktailpeanut/fluxgym#393]
The app just stops as soon as it start to train epoch 1 without any error in console, do you guys have some help to fix that ?

`
...
[2025-04-24 15:42:01] [INFO] override steps. steps for 10 epochs is / 指定エポックまでのステップ数: 1500
[2025-04-24 15:42:01] [INFO] enable fp8 training for U-Net.
[2025-04-24 15:42:01] [INFO] enable fp8 training for Text Encoder.
[2025-04-24 15:42:01] [INFO] INFO     set U-Net weight dtype to     train_network.py:826
[2025-04-24 15:42:01] [INFO] torch.float8_e4m3fn
[2025-04-24 15:42:01] [INFO] INFO     prepare CLIP-L for fp8:  flux_train_network.py:482
[2025-04-24 15:42:01] [INFO] set to
[2025-04-24 15:42:01] [INFO] torch.float8_e4m3fn, set
[2025-04-24 15:42:01] [INFO] embeddings to
[2025-04-24 15:42:01] [INFO] torch.bfloat16
[2025-04-24 15:42:21] [INFO] running training / 学習開始
[2025-04-24 15:42:21] [INFO] num train images * repeats / 学習画像の数×繰り返し回数: 150
[2025-04-24 15:42:21] [INFO] num validation images * repeats / 学習画像の数×繰り返し回数: 0
[2025-04-24 15:42:21] [INFO] num reg images / 正則化画像の数: 0
[2025-04-24 15:42:21] [INFO] num batches per epoch / 1epochのバッチ数: 150
[2025-04-24 15:42:21] [INFO] num epochs / epoch数: 10
[2025-04-24 15:42:21] [INFO] batch size per device / バッチサイズ: 1
[2025-04-24 15:42:21] [INFO] gradient accumulation steps / 勾配を合計するステップ数 = 1
[2025-04-24 15:42:21] [INFO] total optimization steps / 学習ステップ数: 1500
[2025-04-24 15:42:39] [INFO] 2025-04-24 15:42:39 INFO     unet dtype:                  train_network.py:1323
[2025-04-24 15:42:39] [INFO] torch.float8_e4m3fn, device:
[2025-04-24 15:42:39] [INFO] cuda:0
[2025-04-24 15:42:39] [INFO] INFO     text_encoder [0] dtype:      train_network.py:1329
[2025-04-24 15:42:39] [INFO] torch.float8_e4m3fn, device:
[2025-04-24 15:42:39] [INFO] cuda:0
[2025-04-24 15:42:39] [INFO] INFO     text_encoder [1] dtype:      train_network.py:1329
[2025-04-24 15:42:39] [INFO] torch.bfloat16, device: cpu
[2025-04-24 15:42:39] [INFO] steps:   0%|          | 0/1500 [00:00<?, ?it/s]E:\fluxtrain\fluxgym\env\Lib\site-packages\diffusers\configuration_utils.py:141: FutureWarning: Accessing config attribute `num_train_timesteps` directly via 'FlowMatchEulerDiscreteScheduler' object attribute is deprecated. Please access 'num_train_timesteps' over 'FlowMatchEulerDiscreteScheduler's config object instead, e.g. 'scheduler.config.num_train_timesteps'.
[2025-04-24 15:42:39] [INFO] deprecate("direct config name access", "1.0.0", deprecation_message, standard_warn=False)
[2025-04-24 15:42:39] [INFO] 
[2025-04-24 15:42:39] [INFO] epoch 1/10
[2025-04-24 15:42:39] [INFO]
`

**Comment by tjinada** [comment]
I have this same error. Were you able to figure out what the solution is?  Sometimes it eventually goes through ( after waiting for a while), but then get stuck in the next epoch
